Add documentation for Copy-UserInternationalSettingsToSystem cmdlet (#2645)
MicrosoftDocs/windows-powershell-docs: docset/winserver2022-ps/international/Copy-UserInternationalSettingsToSystem.md

---
ms.mktglfcycl: manage
ms.sitesec: library
ms.author: ertorres
author: erik0686
description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
external help file: Microsoft.InternationalSettings.Commands.dll-Help.xml
keywords: powershell, cmdlet

> manager: aandrejs

Reviewer: I hope this isn't permanently stored. These things age poorly.

ms.date: 09/08/2021
ms.prod: w11
ms.technology:
ms.topic: reference
online version: 
schema: 2.1.0
title: Copy-UserInternationalSettingsToSystem
ms.reviewer:
ms.assetid: 
---

# Copy-UserInternationalSettingsToSystem

## SYNOPSIS

> Copies the current user's international settings (Windows Display language, Input language, Regional Format/locale and Location/GeoID) to one or both of the following:

Reviewer 2: ```suggestion
Copies the current user's international settings (Windows Display language, Input language, Regional Format/locale, and Location/GeoID) to one or both of the following:
```

> Copies the current user's international settings (Windows Display language, Input language, Regional Format/locale and Location/GeoID) to one or both of the following:
> * Welcome screen and system accounts
> * New user accounts

Reviewer 2: ```suggestion
Copies the current user's international settings (Windows Display language, Input language, Regional Format/locale, and Location/GeoID) to one or both of the following:
* Welcome screen and system accounts
* New user accounts
```

> This is a system setting. It can only be changed by a user who has Administrator permissions. Changes take effect after the computer is restarted.

Reviewer 2: ```suggestion
This is a system setting. It can only be changed by a user who has Administrator permissions. Changes take effect after the device is restarted.
```

## SYNTAX

```
Copy-UserInternationalSettingsToSystem [-WelcomeScreen <Boolean>] [-NewUser <Boolean>] [<CommonParameters>]
```

## DESCRIPTION
The **Copy-UserInternationalSettingsToSystem** cmdlet gets the **international settings** from the current user.
You can decide whether to copy these settings into the **Welcome screen and system accounts**, the **New user accounts**, or both.

> This cmdlet needs two parameters: **-WelcomeScreen** and **-NewUser**.

Reviewer 3: ```suggestion
This cmdlet takes two parameters: **-WelcomeScreen** and **-NewUser**.
```

> When any of the parameters is set to $True, this gets the current values from the current user and copies them to the system settings for the selected options based on the parameters that were set to $True.

Reviewer 3: ```suggestion
When any of the parameters is set to $True, this cmdlet gets the current values from the current user and copies them to the system settings for the selected options based on the parameters that were set to $True.
```

> **Important:** Please note that this PowerShell Cmdlet is only available for Windows 11 and higher.

Reviewer 2: ```suggestion
**Important:** Note that this PowerShell cmdlet is only available for Windows 11 and later.
```

## EXAMPLES

> ### Example 1: Copy settings into both the Welcome screen and new user accounts

Reviewer 3: ```suggestion
### Example 1: Copy settings into both the Welcome screen and system accounts, and new user accounts
```

```
PS C:\> Copy-UserInternationalSettingsToSystem -WelcomeScreen $True -NewUser $True
```

> This example copies the Windows Display language, Input language, Regional Format/locale and Location/GeoID into both the Welcome screen and the new user accounts.

Reviewer 3: ```suggestion
This example copies the Windows Display language, Input language, Regional Format/locale and Location/GeoID into both the Welcome screen and system accounts, and new user accounts.
```

> This example copies the Windows Display language, Input language, Regional Format/locale and Location/GeoID into both the Welcome screen and system accounts, and new user accounts.

Reviewer 2: ```suggestion
This example copies the Windows Display language, Input language, Regional Format/locale, and Location/GeoID into both the Welcome screen and system accounts, and new user accounts.
```

> ### Example 2: Copy the settings into only the Welcome screen

Reviewer 3: ```suggestion
### Example 2: Copy the settings into only the Welcome screen and system accounts
```

```
PS C:\> Copy-UserInternationalSettingsToSystem -WelcomeScreen $True -NewUser $False
```

> This example copies the Windows Display language, Input language, Format/locale and Regional Location/GeoID to the Welcome screen only.

Reviewer 3: ```suggestion
This example copies the Windows Display language, Input language, Format/locale and Regional Location/GeoID to the Welcome screen and system accounts only.
```

> This example copies the Windows Display language, Input language, Format/locale and Regional Location/GeoID to the Welcome screen and system accounts only.

Reviewer 2: ```suggestion
This example copies the Windows Display language, Input language, Format/locale, and Regional Location/GeoID to the Welcome screen and system accounts only.
```

## PARAMETERS

### -WelcomeScreen

> Copies the settings into the Welcome Screen and system accounts.

Reviewer 2: ```suggestion
Copies the settings into the Welcome screen and system accounts.
```
